docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
We need to change `deletePerson(1)` in line 51 :grin:

Reply:
Changed to deleteModule(1)!